app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarMenu.kt

```diff
@@ -23,7 +23,7 @@ interface ToolbarMenu {
         object InstallPwaToHomeScreen : Item()
         object AddToHomeScreen : Item()
         object SyncedTabs : Item()
-        object SyncAccount : Item()
+        data class SyncAccount(val signedIn: Boolean) : Item()
```

Comment:
I think we can remove this `signedIn` attribute for now since we aren't using it in this patch.

Reply:
We are using it to navigate. See `HomeFragment` and `BrowserToolbarMenuController.kt`